docs/src/components/pricing/PricingTable.tsx

```diff
@@ -18,18 +18,17 @@ const planInfo = {
   community: {
     color: 'green',
     title: 'Community',
-    description:
-      'Get started with the industry-standard UI library for building React user interfaces (open-source, MIT licensed).',
+    description: 'Get started with the industry-standard UI library for building React user interfaces (open-source, MIT licensed).',
```

Comment:
@samuelsycamore I guess we should slightly rephrase this description since it is mostly resonating with Material UI and not Core as a whole. The "Community" plan (practically the same as "no plan" 😅) is the free usage of everything that is MIT, therefore including all Core and the free-tier of the X components.

Reply:
How about something along these lines? The idea is to make it clear that all _components_ are free (right?) but you can upgrade for more _features and support_.

```suggestion
    description: 'All of MUI's component libraries are MIT-licensed, open source, and free forever. Upgrade for more advanced MUI X features and support.',
```

Reply:
I feel like we could ditch the "Upgrade..." part. As the three plans are close to one another on this page, I guess there's no need for a "leading" copy as the content of plans in comparison should probably speak for themselves!

```suggestion
    description: 'Get all of MUI's component libraries—MIT-licensed, open source/open core, and free forever.',
```

How about that?